lang/lua-bit32/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lua-bit32
+PKG_VERSION:=5.3.0
+PKG_RELEASE:=1
+
+
+PKG_SRC_NAME:=lua-compat-5.2
+PKG_SRC_VERSION:=0.3
+PKG_SOURCE_URL=https://codeload.github.com/keplerproject/$(PKG_SRC_NAME)/tar.gz/v$(PKG_SRC_VERSION)?
```

Comment:
get rid of PKG_SRC_NAME. variables are normally not used in URLs unless they change (the version).

Reply:
what requirement is this? where is it documented for contributors?

Reply:
https://github.com/openwrt/packages/commit/f82287cf5c1b2acd46feb52fb9b3cb3697e7086b

I've done this before but was told to replace with normal name.

Reply:
@neheb, I think you are overextending this requirement.  It is only for `call`, `eval` or `define` lines.  Note that f82287c does not touch any other lines, and that 
`git grep '^PKG_SOURCE_URL.*PKG_NAME' f82287c` will return 34 lines that were left unchanged.